src/config.js

```diff
@@ -145,6 +145,7 @@ function defaults() {
       labelLimit: 180,
       labelPadding: 2,
       ticks: true,
+      integer: false,
```

Comment:
I don't think we need or want a configuration entry for this. Why would someone want to make _all_ axes integer only?!

Reply:
Say someone has their own aggregation for a categorical histogram. Specifying an axis to render only integer values is quite useful when we don't have many counts  say (1-4 counts per category). Currently the tick function renders float values for counts that low. tried to use the "format" tag in scales but it didn't change the tick marks properly. Any suggestions to fix this issue?